src/com/ppm/integration/agilesdk/connector/jira/util/JIRACustomIdFinder.java

```diff
@@ -6,23 +7,23 @@
 import org.json.JSONObject;
 
 public class JIRACustomIdFinder {
-	// Different Jira Instance has different customfield Id
-	public static String findId(JSONObject schemas, String customConstant) {
-		Iterator<String> i = schemas.keys();
-		String sprintCutomId = "";
-		while (i.hasNext()) {
-			String key = i.next();
-			try {
-				String custom = schemas.getJSONObject(key).getString("custom");
-				if (customConstant.equals(custom)) {
-					sprintCutomId = key;
-					break;
-				}
-			} catch (JSONException e) {
+    // Different Jira Instance has different customfield Id
```

Comment:
You're using tabs instead of spaces for formatting, no? Please change that in future commits. :)